### ISSUE TITLE
Connect Stats to Ability Scores

### ISSUE THREAD

**Issue by Pyxxil** [Matteas-Eden/roll-for-reaction#72]
**User Story**
As a gamer, I would like the stats displayed to me to accurately reflect the values I chose using the Ability dialog.

**Acceptance Criteria**
- The values I chose from the dialog at the beginning of the game should be displayed in the stats on screen
- The modifier's displayed should be accurate based on these stats as well

**Why is this feature needed? Please describe the problem your requested feature wants to solve**
This is needed because our project relies heavily on these ability scores and their modifiers. Therefore we should be displaying these accurately to the user so that they can rely on them when traversing the dungeon.

**Describe the solution you'd like**
These ability scores are populated with the ability dialog at the start of the game, so the data exists. We also already have the ability to display these values, however that's currently with hard coded values. We just need to change these hard coded values to be taken instead from the data generated by the dialog.

Blocked by #47

**Comment by Pyxxil** [comment]
Superseded by #71
